src/modules/FioAddress/components/ConnectWallets.js

```diff
@@ -2,17 +2,16 @@
 
 import type { EdgeCurrencyWallet } from 'edge-core-js'
 import * as React from 'react'
-import { ActivityIndicator, FlatList, Image, ScrollView, StyleSheet, Switch, TouchableHighlight, View } from 'react-native'
+import { ActivityIndicator, FlatList, Image, ScrollView, Switch, TouchableHighlight, View } from 'react-native'
 import { Actions } from 'react-native-router-flux'
 import { connect } from 'react-redux'
 
 import { showError } from '../../../components/services/AirshipInstance'
+import { type Theme, type ThemeProps, cacheStyles, withTheme } from '../../../components/services/ThemeContext'
 import * as Constants from '../../../constants/indexConstants'
 import s from '../../../locales/strings.js'
-import { THEME } from '../../../theme/variables/airbitz.js'
 import { type RootState } from '../../../types/reduxTypes'
 import type { FioConnectionWalletItem } from '../../../types/types'
-import { scale } from '../../../util/scaling.js'
 import T from '../../UI/components/FormattedText/FormattedText.ui.js'
```

Comment:
do not use T component. use EdgeText instead since it is themed

Reply:
yes, sure, missed that

Reply:
@paullinator you know, it has already changed :)